visualization/app/codeCharta/state/store/fileSettings/blacklist/blacklist.merger.ts

```diff
@@ -17,13 +17,9 @@ export function getMergedBlacklist(inputFiles: CCFile[], withUpdatedPath: boolea
 						: oldBlacklistItem.path,
 					type: oldBlacklistItem.type
 				}
-				const equalBlacklistItems = blacklist.find(b => b.path == blacklistItem.path && b.type == blacklistItem.type)
-
-				if (!equalBlacklistItems) {
-					blacklist.push(blacklistItem)
-				}
+				blacklist.set(`${blacklistItem.path}|${blacklistItem.type}`, blacklistItem)
 			}
 		}
 	}
-	return blacklist
+	return Array.from(blacklist.values())
```

Comment:
Nit: I personally prefer the shorter spread notation:

```suggestion
	return [...blacklist.values()]
```